huaweicloud/services/acceptance/cc/resource_huaweicloud_cc_network_instance_test.go

```diff
@@ -111,24 +111,24 @@ func TestAccNetworkInstance_multiple(t *testing.T) {
 			{
 				Config: testNetworkInstance_multiple(name, 2),
 				Check: resource.ComposeTestCheckFunc(
-					rc.CheckMultiResourcesExists(2),
-					resource.TestCheckResourceAttr("huaweicloud_cc_network_instance.test.0", "type", "vpc"),
-					resource.TestCheckResourceAttr("huaweicloud_cc_network_instance.test.1", "type", "vpc"),
-					resource.TestCheckResourceAttr("huaweicloud_cc_network_instance.test.0", "status", "ACTIVE"),
-					resource.TestCheckResourceAttr("huaweicloud_cc_network_instance.test.1", "status", "ACTIVE"),
-					resource.TestCheckResourceAttrSet("huaweicloud_cc_network_instance.test.0", "domain_id"),
-					resource.TestCheckResourceAttrSet("huaweicloud_cc_network_instance.test.1", "domain_id"),
-					resource.TestCheckResourceAttrPair("huaweicloud_cc_network_instance.test.0", "instance_id",
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr("huaweicloud_cc_network_instance.test1", "type", "vpc"),
+					resource.TestCheckResourceAttr("huaweicloud_cc_network_instance.test2", "type", "vpc"),
+					resource.TestCheckResourceAttr("huaweicloud_cc_network_instance.test1", "status", "ACTIVE"),
+					resource.TestCheckResourceAttr("huaweicloud_cc_network_instance.test2", "status", "ACTIVE"),
```

Comment:
The attribute has no fixed value, how about change it into this:
```suggestion
					resource.TestCheckResourceAttrSet("huaweicloud_cc_network_instance.test2", "status"),
```